modules/appeals_api/app/swagger/appeals_api/v1/api_description.md

```diff
@@ -1,11 +1,14 @@
-The Decision Reviews API allows you to interact with a veteran’s decision reviews, also known as benefit appeals. This API provides a secure and efficient alternative to paper or fax submissions.
+The Decision Reviews API allows you to interact with a Veteran’s decision reviews, also known as benefit appeals. This API provides a secure and efficient alternative to paper or fax submissions and follows the AMA process. To view the status of all decision reviews and benefits appeals submitted according to the legacy benefits appeals process, use the [Appeals Status API](/explore/appeals/docs/appeals?version=current).
 
-> * For more information on the Decision Review process, including the different types of Decision Reviews, please see [this informative page on VA.gov](https://www.va.gov/decision-reviews/).
+Information about the decision reviews process and types of decision reviews is available on the [VA decision reviews and appeals page](https://www.va.gov/decision-reviews/#request-a-decision-review-or-appeal).
 
-> * The Decision Reviews API supports the Appeals Modernization Act (AMA) process only. Appeals going through the legacy Appeal process cannot be managed through this API. Learn more about AMA [here](https://benefits.va.gov/benefits/appeals.asp).
+### Background
+The Decision Reviews API passes data through to Caseflow, a case management system. The API converts decision review data into structured data that can be used for processing and reporting. 
 
-## Technical Summary
+Because this application is designed to allow third-parties to request information on behalf of a Veteran, we are not using VA Authentication Federation Infrastructure (VAAFI) headers or Single Sign On External (SSOe).
 
-### Authorization
+### Authorization and Access
+To gain access to the decision reviews API you must [request an API Key](/apply). API requests are authorized through a symmetric API token which is provided in an HTTP header named `apikey`.
 
-API requests are authorized through a symmetric API token, provided in an HTTP header with name `apikey`.
+### Data refresh
+The Lighthouse Sandbox environment points to the Caseflow UAT environment which refreshes daily. The data in this environment is mock data and does not contain any PII.
```

Comment:
UAT doesn't refresh daily
https://dsva.slack.com/archives/C2ZAMLK88/p1596053539041000?thread_ts=1596053120.040900&cid=C2ZAMLK88

Reply:
confirmed that UAT's DB is never reset
https://dsva.slack.com/archives/C2ZAMLK88/p1596123119046300?thread_ts=1596053120.040900&cid=C2ZAMLK88